docs/onclusterlayering-quickstart.md

```diff
@@ -0,0 +1,492 @@
+# On-Cluster Layering Quickstart Guide
+
+## Prerequisites
+This quick-start guide assumes you have the following:
+- Access to an OpenShift 4.16+ cluster with TechPreview mode enabled.
```

Comment:
Being picky to keep the working consistent with the end user docs. 
```suggestion
- Access to an OpenShift 4.16+ cluster with TechPreview feature gate enabled.
```
Do we need a link to [Enabling features using feature gates](https://docs.openshift.com/container-platform/4.16/nodes/clusters/nodes-cluster-enabling-features.html#nodes-cluster-enabling).